common-util/src/main/java/org/sunbird/common/request/RequestValidator.java

```diff
@@ -138,6 +141,53 @@ public static void validateOrg(Request request) {
     }
   }
 
+
+  public static void validateCreateFirstRootOrg(Request request) {
```

Comment:
Write a separate request validator. You can refer BadgeClassValidator for reference.